cli/cmd/compliance_azure.go

```diff
@@ -68,6 +68,33 @@ Then, select one GUID from an integration and visualize its details using the co
 		},
 	}
 
+	// complianceAzureListTenantsCmd represents the list-tenants sub-command inside the azure command
+	complianceAzureListTenantsCmd = &cobra.Command{
+		Use:     "list-tenants",
+		Aliases: []string{"list"},
+		Short:   "list subscriptions from tenant",
+		Long:    `List all Azure Tenants.`,
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			response, err := cli.LwApi.Compliance.ListAzureTenants()
```

Comment:
Interesting way to do this but, I'm afraid it doesn't follow the standard we put in compliance for AWS. Also, we are creating "an API that doesn't exist in the platform itself" and that goes against the principle to match 100% the Lacework API. I think we should add this functionality to the CLI alone, just like the command to list AWS accounts.